src/patterns/start-using-a-service/index.md.njk

```diff
@@ -0,0 +1,75 @@
+---
+title: Start using a service
+description: Create a starting point for your digital service on GOV.UK
+section: Patterns
+theme: Help users to…
+aliases: start page, start pages
+backlog_issue_id: 111
+layout: layout-pane.njk
+---
+
+GOV.UK services must start on a GOV.UK content page. This links your service to the rest of GOV.UK, and means that users can find it using search or through GOV.UK’s navigation.
+
+## When to use this pattern
+
+Use this pattern to help you prototype a start point for your service, so you can test as much of the user journey as possible.
+
+The public-facing version of your service will need to start on a GOV.UK content page, published using one of the GOV.UK publishing apps. You’ll usually need to agree content for this GOV.UK content page with the GDS content team. Read [guidance on how to get your service on GOV.UK](https://www.gov.uk/service-manual/service-assessments/get-your-service-on-govuk).
+
+## How it works
+
+A service's start point should:
+- give the user just enough information to help them understand what the service does and whether it will meet their need — including [giving the service a name that reflects the problem it solves for users](https://www.gov.uk/service-manual/design/naming-your-service)
+- include a ‘button’ linking into the service, with text that’s consistent with the action you’re asking users to take — for example, ‘Start now’, ‘Sign in’ or ‘Register or update your details’ (if you need a secondary call to action, use a standard link)
```

Comment:
Should this be a 'start button'? Also, does it need to be in quotes?

```suggestion
- include a ‘start button’ linking into the service, with text that’s consistent with the action you’re asking users to take — for example, ‘Start now’, ‘Sign in’ or ‘Register or update your details’ (if you need a secondary call to action, use a standard link)
```

Reply:
Also, can it be a link? 😁

Reply:
The purpose of this line is to get teams to be more open-minded than just saying "Start now" as default so 'button' would works best. The quotes help show it as a component of a start point.

I think they need to be buttons in the CMSs?